index.js

```diff
@@ -31,9 +31,9 @@ function clangFormat(file, enc, style, done) {
 function spawnClangFormat(args, done, stdio) {
   // WARNING: This function's interface should stay stable across versions for the cross-version
   // loading below to work.
-  if (args.indexOf('-version') !== -1 || args.indexOf('--version') !== -1) {
+  if (args.indexOf('--version') !== -1) {
```

Comment:
I removed intercepting of `-version` so that it could be passed onto the native binary (enabling getting it's version), and so that all intercepted commands would be the double-dash ones.

It's not very obvious at all without reading the readme though, and it is a breaking change. This is confined to the last commit so it should be easy to remove from the PR. Do you want me to do so?